examples/oci-image-verification/main.go

```diff
@@ -408,6 +423,29 @@ func getVerificationMaterialTlogEntries(manifestLayer *v1.Descriptor) ([]*protor
 	}, nil
 }
 
+func getVerificationMaterialTimestampEntries(manifestLayer *v1.Descriptor) (*protobundle.TimestampVerificationData, error) {
+	// 1. Get the bundle annotation
+	ts := manifestLayer.Annotations["dev.sigstore.cosign/rfc3161timestamp"]
+	// 2. Get the key/value pairs maps
+	var keyValPairs map[string]string
+	err := json.Unmarshal([]byte(ts), &keyValPairs)
+	if err != nil {
+		return nil, fmt.Errorf("error unmarshaling JSON blob into key/val map: %w", err)
+	}
+	// 3. Verify the key "SignedRFC3161Timestamp" is present
+	if _, ok := keyValPairs["SignedRFC3161Timestamp"]; !ok {
+		return nil, errors.New("error getting SignedRFC3161Timestamp from key/value pairs")
+	}
+	// 4. Construct the timestamp entry list
+	return &protobundle.TimestampVerificationData{
+		Rfc3161Timestamps: []*protocommon.RFC3161SignedTimestamp{
+			{
+				SignedTimestamp: []byte(keyValPairs["SignedRFC3161Timestamp"]),
```

Comment:
I think you would need to do `base64.StdEncoding.DecodeString(keyValPairs["SignedRFC3161Timestamp"]` here.

Reply:
The json data in the manifest should be base64 decoded, but the protobundle struct expects the raw bytes.

Reply:
changed - and it seemed to help, thanks! 😄 , no longer having the asn.1 parsing error (tracing a different one instead - `failed to verify leaf certificate: leaf certificate verification failed`)